pkg/controller/kfservice/resources/knative/configuration.go

```diff
@@ -126,13 +129,13 @@ func (c *ConfigurationBuilder) CreateKnativeConfiguration(name string, metadata
 
 func configurationAnnotationFilter(annotationKey string) bool {
 	switch annotationKey {
-	case autoscaling.TargetAnnotationKey:
-		return true
-	case autoscaling.ClassAnnotationKey:
-		return true
-	case constants.KFServiceGKEAcceleratorAnnotationKey:
-		return true
-	default:
+	case autoscaling.MinScaleAnnotationKey:
+		return false
+	case autoscaling.MaxScaleAnnotationKey:
+		return false
+	case constants.ModelInitializerSourceUriInternalAnnotationKey:
 		return false
```

Comment:
we need to block passing down kubernetes built in `kubectl.kubernetes.io/last-applied-configuration` annotation

Reply:
Done.